packages/components/src/components/dialog/dialog.stories.ts

```diff
@@ -57,271 +67,73 @@ export const Default = {
-                const footer = dialog.querySelector('sd-button[slot="footer"]');
-      
-                footer.addEventListener('click', () => dialog.hide());
-          </script>`
-          }
-        ]
-      })}
-    </div>`;
-  }
-};
-
-/**
- * This shows sd-dialog’s height will never exceed that of the viewport. As such, sd-dialogs will not scroll with the page ensuring the contents are always accessible to the user. It is recomended to use `sd-scrollable` with an enabled shadow property for scrolling content in the `default` slot.
- */
-
-export const Scrolling = {
-  render: (args: any) => {
-    return html` <div style="height: 100vh;">
-      ${generateTemplate({
-        args,
-        constants: [
-          {
-            type: 'slot',
-            name: 'default',
-            value: `<sd-scrollable orientation='vertical' shadows class="w-full"><div class="slot slot--border slot--background slot--text" style="height:150vh; width: 100%; padding: 1rem; justify-content:start;">Scroll down and give it a try!</div></sd-scrollable>`
-          }
-        ]
-      })}
-    </div>`;
-  }
-};
-
-/**
- *  This sample displays `sd-dialog` with two `sd-button` components in the footer slot. When incorporating multiple `sd-buttons`, it is recommended to use distinct button variants for clarity and consistency in user interactions.
- * 
- * You can use the CSS part `sd-dialog::part(footer)` to customize button layout based on your requirements. For instance, you may opt for columns instead of rows for mobile devices, and so on. To achieve that we used the additional style:
- * 
- * ```css
- * 
- *   @media (max-width: 414px) {
-          sd-dialog::part(footer){
-            flex-direction: column;
- *   }
- * }
-
- * ```
- * 
- * 
+ * Use the `open` attribute to show the dialog.
  */
 
-export const ExtendedFooter = {
-  name: 'Sample: Extended Footer',
-  parameters: {
-    controls: {
-      disable: true
-    }
-  },
-  render: () => {
-    return html`
-      <style>
-        @media (max-width: 414px) {
-          sd-dialog::part(footer) {
-            flex-direction: column;
-          }
-        }
-      </style>
-
-      <div style="height: 40vh;">
-        <sd-dialog open="" id="extended-footer"
-          ><p class="sd-paragraph">
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nibh justo ullamcorper odio tempor molestie
-            phasellus dui vel id. Velit in sed.
-          </p>
-          <h4 slot="headline" class="sd-headline sd-headline--size-3xl">Lorem Ipsum</h4>
-          <sd-button variant="secondary" slot="footer" class="w-full">Label</sd-button>
-          <sd-button slot="footer" class="w-full">Label</sd-button>
-        </sd-dialog>
-      </div>
-    `;
-  }
+export const Open = {
+  name: 'Open',
+  render: () => html`
+    <div class="h-[40vh]">
+      <sd-dialog open>
+        <h4 slot="headline" class="sd-headline sd-headline-3xl">Lorem ipsum</h4>
```

Comment:
```suggestion
        <h4 slot="headline" class="sd-headline sd-headline-3xl">Headline</h4>
```

Reply:
@karlbaumhauer why do you suggest this change? It's approved by design to use lorem ipsum in this case.

Reply:
After checking with @coraliefeil, we will not update.